Text/PresidioPIIAnonymization/notebooks/.ipynb_checkpoints/PII Anonymization-checkpoint.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
This file should be removed, it's just a jupyter checkpoint file

Reply:
In the text summarization it is submitted. But I don't mind removing it.